src/Aspire.Hosting/Utils/PasswordUtil.cs

```diff
@@ -6,4 +6,47 @@ namespace Aspire.Hosting.Utils;
 internal static class PasswordUtil
 {
     internal static string EscapePassword(string password) => password.Replace("\"", "\"\"");
+
+    /// <summary>
+    /// Returns a random password of length <paramref name="length"/> that does not contain any characters that would be escaped by <see cref="EscapePassword(string)"/>.
+    /// </summary>
+    /// <remarks>If <paramref name="length"/> is greater than or equal to four, the password is guaranteed to contain an uppercase ASCII letter, a lowercase ASCII letter, a digit, and a symbol.</remarks>
+    internal static string GeneratePassword(int length = 20)
```

Comment:
How cryptographically secure is this implementation? Writing a good password generator is probably more complicated than writing own cache.
There are quite a few related Qs on the web, e.g., https://stackoverflow.com/questions/54991/generating-random-passwords.

Also, having this implementation static essentially forces it for everyone. Shouldn't it be instead be injectable and replaceable?